moneo_config.json

```diff
@@ -0,0 +1,28 @@
+{
+    "exporter_conf":{
```

Comment:
Every other node uses "_config". Is there a good reason to be inconsistent here?

Reply:
let me address

Reply:
addressed